api/compression.md

```diff
@@ -20,5 +20,8 @@ decompress the chunk(s) first.
 
 Starting with TimescaleDB 2.1, users have the ability to modify the schema
 of hypertables that have compressed chunks.
-Specifically, you can add columns to and rename existing columns of 
-such compressed hypertables.
\ No newline at end of file
+Specifically, you can add columns to and rename existing columns of
+such compressed hypertables.
+
+[blog-compression]: https://blog.timescale.com/blog/building-columnar-compression-in-a-row-oriented-database/
+[using-compression]: /how-to-guides/compression/index/
```

Comment:
This will resolve to `/api/[whatever version is selected]/how-to-guides/compression/index` but that page doesn't exist.
I think youre looking for it to link to [this page](https://docs-dev.timescale.com/docs-compression-404-lana/timescaledb/compression-404-lana/how-to-guides/compression/). If thats the case the link needs to be:
`/timescaledb/:currentVersion:/how-to-guides/compression/`

Reply:
```suggestion
[using-compression]: /timescaledb/latest/how-to-guides/compression/index/
```

??

Reply:
@tedsczelecki ??